src/mkdocs_include_markdown_plugin/process.py

```diff
@@ -117,12 +117,34 @@
 MARKDOWN_HTML_IMAGE_REGEX = re.compile(
     r'''
         <img
-        \s+
+        [^>]*
```

Comment:
This doesn't looks good because patterns like `<img my-attr="3>2"  src="..."` will not be matched.

Reply:
Thank you for comment. Update it with more resilient one and add more adversarial test.